pandas/core/generic.py

```diff
@@ -1671,6 +1672,13 @@ def to_json(self, path_or_buf=None, orient=None, date_format=None,
 
             .. versionadded:: 0.21.0
 
+        index : boolean, default True
+            Whether to include the index values in the JSON string. A
+            ValueError will be thrown if index is False when orient is not
```

Comment:
I would say something like "Not including the index (`index=False`) is only supported for orient 'split' and 'table'" instead